nvidia-bert/docker/Dockerfile

```diff
@@ -1,122 +1,10 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+FROM onnxruntime-pytorch-for-examples
```

Comment:
This doesn't seem to be publicly available
When I try `docker pull onnxruntime-pytorch-for-examples` I get the following error:
`Error response from daemon: pull access denied for onnxruntime-pytorch-for-examples, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`

Reply:
Yes, it would be best of this was available as a public image. An image exists and it would be easy to push to MCR with approval. Referenced here is if Dockerfile built from the root of the repository.

Reply:
If onnxruntime-pytorch-for-examples is not public, users cant do `docker build` to build themselves locally. They will have to use a published image (`docker pull`), which might not be ideal depending on the size of the image or for local experimentation, which is the most likely use case for this repo

Reply:
Sorry, perhaps I wasn't clear. "onnxruntime-pytorch-for-examples" _is_ the image on building the included Dockerfile above, the one in the root of the repository. The tag comes if you use the included docker/build.sh. The Dockerfile is based on the NVIDIA CUDA 10.2 runtime image. So they should be able to build themselves. Seems OK? Perhaps we need to add some clarity in the documentation?

Reply:
After meeting discussion with others, we are going to create a public MCR image different from the one generated by the Dockerfile in this PR. It will be based on an image created by AML team (derived itself from a minimal Dockerfile by Jingyan in onnxruntime repository). Therefore I will wait for its release/test. For more immediate needs, I can be contacted for working image.

Reply:
Published as mcr.microsoft.com/azureml/onnxruntime-training:0.1-rc2-openmpi4.0-cuda10.2-cudnn7.6-nccl2.7.6